src/main/java/seedu/address/logic/commands/AddInterviewDateCommand.java

```diff
@@ -74,9 +74,10 @@ private static InternshipApplication createInternshipWithInterviewDate(
         CompanyName companyName = internshipToAddInterviewDate.getCompanyName();
         JobTitle jobTitle = internshipToAddInterviewDate.getJobTitle();
         Contact contact = internshipToAddInterviewDate.getContact();
+        boolean archived = internshipToAddInterviewDate.archived();
         InternshipStatus status = internshipToAddInterviewDate.getStatus();
 
-        return new InternshipApplication(companyName, jobTitle, contact, status, interviewDate);
+        return new InternshipApplication(companyName, jobTitle, contact, status, archived, interviewDate);
```

Comment:
A better naming for this is probably `isArchived`, but this is not a huge issue I would say.